presidio-analyzer/.dockerignore

```diff
@@ -5,6 +5,8 @@
 !model*
 !setup*
 !requirements.txt
+!requirements-dev.txt
```

Comment:
The test fails because it can't find requirements-dev.txt 
We should have this as a part of the dockerignore, it is required for running tests